src/antelope/types/Actions.ts

```diff
@@ -22,256 +23,266 @@ export interface GetActionsResponse {
 }
 
 export interface Account {
-  account: string;
-  authorization: Authorization[];
-  data: unknown;
-  name: string;
+    account: string;
+    authorization: Authorization[];
+    data: unknown;
+    name: string;
 }
 
 export interface Authorization {
-  actor: string;
-  permission: string;
+    actor: string;
+    permission: string;
 }
 
 interface Resource {
-  used: number;
-  available: number;
-  max: number;
+    used: number;
+    available: number;
+    max: number;
 }
 
 export type AccountDetails = {
-  account: {
-    account_name: string;
-    core_liquid_balance: string;
-    cpu_limit: Resource;
-    cpu_weight: number;
-    created: string;
-    net_limit: Resource;
-    net_weight: number;
-    permissions: Permission[];
-    privileged: boolean;
-    ram_quota: number;
-    ram_usage: number;
-    refund_request: Refund;
-    rex_info: null | {
-      matured_rex: string;
-      vote_stake: string;
-      rex_balance: string;
-      rex_maturities: Maturities[];
+    account: {
+        account_name: string;
+        core_liquid_balance: string;
+        cpu_limit: Resource;
+        cpu_weight: number;
+        created: string;
+        net_limit: Resource;
+        net_weight: number;
+        permissions: Permission[];
+        privileged: boolean;
+        ram_quota: number;
+        ram_usage: number;
+        refund_request: Refund;
+        rex_info: null | {
+            matured_rex: string;
+            vote_stake: string;
+            rex_balance: string;
+            rex_maturities: Maturities[];
+        };
+        subjective_cpu_bill_limit: Resource;
+        total_resources: {
+            owner: string;
+            net_weight: string;
+            cpu_weight: string;
+            ram_bytes: number;
+        };
+        voter_info: {
+            owner: string;
+            proxy: string;
+            producers: string[];
+            staked: number;
+            last_stake: number;
+            last_vote_weight: string;
+            proxied_vote_weight: number;
+            is_proxy: number;
+        };
+        self_delegated_bandwidth: { net_weight: string; cpu_weight: string };
     };
-    subjective_cpu_bill_limit: Resource;
-    total_resources: {
-      owner: string;
-      net_weight: string;
-      cpu_weight: string;
-      ram_bytes: number;
-    };
-    voter_info: {
-      owner: string;
-      proxy: string;
-      producers: string[];
-      staked: number;
-      last_stake: number;
-      last_vote_weight: string;
-      proxied_vote_weight: number;
-      is_proxy: number;
-    };
-    self_delegated_bandwidth: { net_weight: string; cpu_weight: string };
-  };
-  actions: Action[];
-  links: string[];
-  query_time_ms: number;
-  tokens: Token[];
-  total_actions: number;
+    actions: Action[];
+    links: string[];
+    query_time_ms: number;
+    tokens: Token[];
+    total_actions: number;
 };
 
+export interface TokenBalance {
+    amount: number; // do we need this?
```

Comment:
resolve / delete comment. is `amount` used?

Reply:
I think we should talk about this
https://github.com/telosnetwork/telos-wallet/issues/280